pom.xml

```diff
@@ -26,7 +26,7 @@ THE SOFTWARE.
   <parent>
     <groupId>org.jenkins-ci.plugins</groupId>
     <artifactId>plugin</artifactId>
-    <version>2.6</version>
+    <version>2.22</version>
```

Comment:
2.23 led to errors due to a `TestExtension` usage in `workflow-support`'s tests, so I bumped back down to 2.22 for now at least.